client/test/unit/lib/solana/solanaBorsh.test.ts

```diff
@@ -79,8 +82,11 @@ describe('solanaBorsh', function () {
   });
 
   it('should serde AssignableI32', function () {
-    range(0, 10).forEach(() => {
-      const value = randomInt(-(2 ** 31), 2 ** 31);
+    const max = 157;
+    range(0, max).forEach((n) => {
+      const value = Math.floor(
+        (n * (I32_BOUNDS.max - I32_BOUNDS.min)) / max + I32_BOUNDS.min
```

Comment:
Does Bound.max - Bound.min run out of bounds? Seems like your ok because number is 64 bit.

Reply:
Yes, the max safe integers for an f64 are on the order of 2^53